tests/slurm_plugin/test_resume.py

```diff
@@ -469,20 +439,14 @@ def test_resume_launch(
                     call(nodeset, reason=f"(Code:{error_code})Failure when resuming nodes")
                 )
             mock_handle_failed_nodes.assert_has_calls(mock_handle_failed_nodes_calls)
-            if job_level_scaling:
-                mock_terminate_instances.assert_called_with(ANY, mock_resume_config.terminate_max_batch_size)
-            else:
-                mock_terminate_instances.assert_not_called()
+            mock_terminate_instances.assert_called_with(ANY, mock_resume_config.terminate_max_batch_size)
```

Comment:
is expected that now this is always called?

Reply:
Answer: yes, because now the instances not assigned are terminated at the end, previously the instances were assigned "at best-effort".